tools/src/main/groovy/cz/o2/proxima/tools/groovy/internal/ProximaInterpreter.groovy

```diff
@@ -35,11 +38,13 @@ public class ProximaInterpreter extends Interpreter {
 
   private final AtomicInteger scriptNo = new AtomicInteger()
 
-  public ProximaInterpreter(
+  ProximaInterpreter(
       final ClassLoader classLoader,
-      final Binding binding) {
+      final Binding binding,
+      final CompilerConfiguration configuration) {
 
-    super(classLoader, binding, null);
+    super(classLoader, binding, configuration);
+    Preconditions.checkState(this.shell.getClassLoader() instanceof ToolsClassLoader);
```

Comment:
```suggestion
    Preconditions.checkState(getShell.getClassLoader() instanceof ToolsClassLoader);
```

Reply:
I think this check needs to be removed as it introduces chicken-egg problem :)

- java code needs groovy code to compile
- groovy code needs a java code to compile